### ISSUE TITLE
Show haproxy starutp errors in container log

### ISSUE THREAD

**Issue by EugenMayer** [rancher/rancher#5584]
It is possible to add a wrong key to a certificate. Even though it is a pretty nasty mistake by the user, we can simply validate the private key / certificate

```
hash_cert=`openssl x509 -noout -modulus -in certificate.crt | openssl md5`
hash_key=`openssl rsa -noout -modulus -in privateKey.key | openssl md5`
if [ $hash_cert != $hash_key ]; then echo "does not match"; exit 1;fi
exit0
```

or similar. Not sure we can validate this in JS, there are some node packages offering that, if the idea would be doing this in the UI.

Currently when doing this mistake and using a broken cert/key combinatiuon in a load balancer service, the load balance will get stuck in 'initialize' and not showing anything useful in the logs. See https://github.com/rancher/rancher/issues/5563 for an example ( of my stupidity )

**Comment by vincent99** [comment]
We had validation for this before but ran into endless edge cases where Java PKI handles it slightly differently than OpenSSL which handles it slightly differently than HAProxy, which resulted in much more false-positives than it was worth claiming things are not valid that would have actually worked.

Showing the error in the container log is a fair point, @alena1108 can you make sure it does that for new lb?

**Comment by alena1108** [comment]
@vincent99 all haproxy errors will be available via container logs for new lb controller container.

**Comment by sangeethah** [comment]
Tested with latest build from master:
Lb container's logs show haproxy error messages now (Tested by starting a LB container with invalid values in config parameter) . 

There is an issue relating to log level not being correct which is tracked in https://github.com/rancher/rancher/issues/6504.

@EugenMayer , this fix will be available in 1.2 release.
